src/browser/api/application.js

```diff
@@ -343,16 +345,18 @@ Application.getShortcuts = function(identity, callback, errorCallback) {
 };
 
 Application.getInfo = function(identity, callback) {
-    const app = Application.wrap(identity.uuid);
-    const manifestObj = coreState.getManifest(identity);
-    const { url: manifestUrl, manifest } = manifestObj;
+    const app = coreState.appByUuid(identity.uuid);
+
+    const manifestObj = coreState.getClosestManifest(identity);
+    const { url, manifest } = manifestObj || {};
 
-    const parentApp = coreState.getAppAncestor(identity.uuid);
-    const parentUuid = (parentApp.uuid === identity.uuid) ? null : parentApp.uuid;
+    const parentUuid = Application.getParentApplication(identity);
+    const launchMode = app && app.appObj && app.appObj.launchMode;
```

Comment:
This was before a string or `undefined`, now can also be `false`... ```¯\_(ツ)_/¯```

Reply:
I believe it should only have returned a string before (at least according to docs), I think this is unnecessary protection.  The call was erroring out in the core (due to [this](https://github.com/HadoukenIO/core/pull/449)) so I added it when trying to determine why and kept it in as an unnecessary failsafe...